zwave-classifier.js

```diff
@@ -656,6 +721,26 @@ class ZWaveClassifier {
     }
   }
 
+  addCarbonMonoxideLevelProperty(node, carbonMonoxideLevelValueId) {
+    const descr = {
+      '@type': 'CarbonMonoxideLevelProperty',
+      label: 'Carbon Monoxide Level',
+      type: 'number',
+      readOnly: true,
+    };
+    descr.unit = 'ppm';
+    this.addProperty(
+      node,
+      'carbonMonoxideLevel',
+      descr,
+      carbonMonoxideLevelValueId,
+    );
+
+    if (!node['@type'].includes('CarbonMonoxideSensor')) {
+      node['@type'].push('CarbonMonoxideSensor');
+    }
+  }
```

Comment:
The above if should be removed (or probably commented out) as we don;t yet have a CarbonMonoxideSensor @type yet. There is a proposal to add one, but currently we just have the Alarm @type.

Reply:
This is for the the multilevel property of carbon monoxide level. If I do not add a property, we fail with the following:

```
  ● Fibaro-FGSD001-CO-Sensor.json › properties

    Failed: "Missing property carbonMonoxideLevel"

      54 |           expect(property.asDict()).toEqual(jsonProperty);
      55 |         } else {
    > 56 |           fail(`Missing property ${propertyName}`);
         |           ^
      57 |         }
      58 |       });
      59 |

      at Env.fail (node_modules/jest-jasmine2/build/jasmine/Env.js:778:61)
      at fail (test/classifier-test.js:56:11)
          at Map.forEach (<anonymous>)
      at Object.forEach (test/classifier-test.js:50:23)

```